Lib/venv/__init__.py

```diff
@@ -303,8 +303,11 @@ def setup_python(self, context):
             copier(context.executable, path)
             if not os.path.islink(path):
                 os.chmod(path, 0o755)
-            for suffix in ('python', 'python3',
-                           f'python3.{sys.version_info[1]}'):
+
+            suffixes = ['python', 'python3', f'python3.{sys.version_info[1]}']
+            if sys.version_info[:2] == (3, 14):
+                suffixes.append('pythonπ')
```

Comment:
From @ambv , this should use the mathematical symbol variant of 𝜋, rather than the Greek letter variant:

```suggestion
                suffixes.append('python𝜋')
```

Reply:
@hauntsaninja Let's include this, remove the NEWS, and keep the version check. Thanks!

Reply:
Thanks, I've made the updates!

(Note that U+1d70b isn't NFKC normalised. I know in some cases non-normalised Unicode can cause issues in filesystems, but I think that's usually about NFC/NFD normalisation)